notebooks/advanced_graphs.ipynb

```diff
@@ -25,7 +25,7 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": true
+    "collapsed": false
```

Comment:
There seems to be a missing proto compile/copy as used in other notebooks. One gets the error:
ImportError                               Traceback (most recent call last)
<ipython-input-11-d0d0a65d288d> in <module>()
      1 import requests
      2 from requests.auth import HTTPBasicAuth
----> 3 from proto import prediction_pb2
      4 from proto import prediction_pb2_grpc
      5 import grpc

ImportError: cannot import name prediction_pb2

Reply:
Did you build the protos locally first?

Reply:
I always add to the notebooks: 

!cp ../proto/prediction.proto ./proto
!python -m grpc.tools.protoc -I. --python_out=. --grpc_python_out=. ./proto/prediction.proto

so they are self contained

Reply:
Added